app/app.go

```diff
@@ -169,7 +168,7 @@ func NewHeimdallApp(
 		invCheckPeriod:    invCheckPeriod,
 		keys:              keys,
 		tkeys:             tkeys,
-		memKeys:           memKeys,
+		// memKeys:           memKeys,
```

Comment:
I would suggest to add a TODO if we gonna use them in future. If we are planning to enable IBC, we would need this for capability module
```suggestion
		// memKeys:           memKeys,
```

Reply:
I will remove `memKeys` through out the code. We don't need it.

Reply:
@anilCSE Fixed and pushed. Please review.